software/rfcs/rocketCtl.md

```diff
@@ -0,0 +1,68 @@
+# RocketCtl
+
+RocketCtl or RocketControl is our system for configuring Apex systems before launch. It can control much of the setup of the avionics without needing us to recompile the code. 
+
+## Pre-reading
+
+1. Knowing that UART = Serial comminication = sending data on a wire to the avionics.
+
+2. What an RPC is.
```

Comment:
"RPC is an international commercial law firm headquartered in London, United Kingdom. It is a UK top 50 law firm and has been named Law Firm of the Year three times since 2014, and Best Legal Adviser every year since 2009."

Reply:
Exactly

https://www.techtarget.com/searchapparchitecture/definition/Remote-Procedure-Call-RPC?amp=1